cg/services/bcl_convert_metrics_service/bcl_convert_metrics_service.py

```diff
@@ -77,3 +79,35 @@ def _create_bcl_convert_sequencing_metrics(
             sample_base_mean_quality_score=mean_quality_score,
             created_at=datetime.now(),
         )
+
+    @staticmethod
+    def create_sample_run_metrics(
+        sample_internal_id: str,
+        lane: int,
+        run_metrics_id: int,
+        metrics_parser: MetricsParser,
+        store: Store,
+    ) -> IlluminaSampleRunMetrics:
```

Comment:
There are a lot of parameters here which to me is always a hint that something can be simplified. Do the metrics_parser and store need to be passed?